assets/js/quiz_page.js

```diff
@@ -0,0 +1,57 @@
+// hiding all questions
+
+function hideAll () {
+    $('.questionOne, .questionTwo, .questionThree, .buttonOne, .buttonTwo, .buttonThree').hide();
```

Comment:
can we improve naming here? for something more descriptive, for example: caloriesInput, carbsInput ?